forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt

```diff
@@ -0,0 +1,14 @@
+Name:Yes Man, Personal Securitron
+ManaCost:2 W
+Types:Legendary Artifact Creature Robot
+PT:2/2
+A:AB$ GainControl | Cost$ T | Defined$ Self | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | SubAbility$ DBImmediateTrigger | PlayerTurn$ True | StackDescription$ SpellDescription | SpellDescription$ Target opponent gains control of CARDNAME. When they do, you draw two cards and put a quest counter on NICKNAME.
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Self | ConditionPresent$ Card.ControlledBy ParentTarget | Execute$ DBDraw | StackDescription$ None
```

Comment:
`TriggerDescription$ When they do, you draw two cards and put a quest counter on NICKNAME.`